src/components/Pages/reserve/Reserve.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import './style.css';
+import { useForm } from 'react-hook-form';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { createReservation } from '../../../redux/reservations/reserve';
+
+const ReservationForm = () => {
+  const dispatch = useDispatch();
+  const { data } = useSelector((state) => state.MostRecent);
+  const navigate = useNavigate();
+  /* eslint-disable */
```

Comment:
Please avoid this. If you are facing any issue dealing with the linters, you should find a way out or contact one of us to help you. This doesn't make the code looks professional. Kindly fix this. 👍🏾

Reply:
okay i will fix it